src/compiler/commandLineParser.ts

```diff
@@ -446,6 +446,9 @@ namespace ts {
                             }
                             if (opt.isFilePath) {
                                 value = normalizePath(combinePaths(basePath, value));
+                                if (value === "") {

```

Comment:
i do not think this is the right place for this change. normalizePath is expected to remove all "." and ".." s.